technic/machines/supply_converter.lua

```diff
@@ -181,9 +181,11 @@ minetest.register_node("technic:supply_converter", {
 		"technic_supply_converter_side.png"
 		},
 	groups = {snappy=2, choppy=2, oddly_breakable_by_hand=2,
-		technic_machine=1, technic_all_tiers=1},
+		  technic_machine=1, technic_all_tiers=1, axey=2, handy=1},
```

Comment:
Minor style / indent
```suggestion
		technic_machine=1, technic_all_tiers=1, axey=2, handy=1},
```